src/components/profile/profile.jsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import Button from '@material-ui/core/Button';
+import { makeStyles } from '@material-ui/core';
+import { Link } from 'react-router-dom';
+
+const Profile = () => {
+    const useStyles = makeStyles((theme) => ({
+        root: {
+            '& > *': {
+                margin: theme.spacing(1),
+            },
+        }, testButtons: {
+            '&': {
+                margin: 40,
+            }
+        },
+    }));
+    const classes = useStyles();
+
+    return <div>
+
+        <div className='routingButtons'>
+            <Button variant="contained" color="primary" disabled className={classes.testButtons}>
+                <Link to='/login'>Login</Link>
+            </Button>
+            <Button variant="contained" color="primary" disableElevation className={classes.testButtons}>
+                <Link to='/counter'>Counter</Link>
+            </Button>
+            <Button variant="contained" color="primary" disableElevation className={classes.testButtons}>
+                <Link to='/buttons'>Buttons</Link>
+            </Button>
+
+        </div>
+    </div>;
```

Comment:
why do you need the div? you have one inside

Reply:
https://github.com/mtalstykh-exadel/team4_frontend/blob/27f79d12b2ded8e5b31bd2b44a4a2d0ffd8adfa8/src/components/profile/profile.jsx#L28-L31